api/handler/my_session/delete_current_session.go

```diff
@@ -0,0 +1,67 @@
+package mysession
+
+import (
+	"encoding/json"
+	"net/http"
+	"userland/api/helper"
+	"userland/api/jwt"
+	"userland/api/response"
+	"userland/store"
+)
+
+func DeleteCurrentSession(userStore store.UserStore, tokenStore store.TokenStore) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		_, err := helper.AuthenticateUserAccessToken(r, tokenStore)
+		if err != nil {
+			w.Header().Set("Content-Type", "application/json")
+			w.WriteHeader(http.StatusUnauthorized)
+			json.NewEncoder(w).Encode(response.Unautorized_request(err.Error()))
+			return
+		}
+
+		// getting the jwt id for access token and refresh token
+		atJti, _, err := jwt.GetAtJtiNRtJtiFromAccessToken(r)
+		if err != nil {
+			w.Header().Set("Content-Type", "application/json")
+			w.WriteHeader(http.StatusUnauthorized)
+			json.NewEncoder(w).Encode(response.Unautorized_request(err.Error()))
+			return
+		}
+
+		// removing the current access token in redis
+		deleted, err := jwt.DeleteATAuth(atJti, tokenStore)
```

Comment:
this and remove refresh token are done better last. So if any problem were to occur during, say `DeleteCurrentSession`, user can still use the same token to retry the operation. Similar comment to `delete_other_session.go`